src/renderer/data/Config/configState.ts

```diff
@@ -1,9 +1,9 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { stat } from 'fs';
```

Comment:
I presume this is unused? as we're on the client side, so shouldn't be accessing the fs?

Reply:
good point, this must be VS code gone bad